skyportal/handlers/api/gwdetector.py

```diff
@@ -0,0 +1,234 @@
+from marshmallow.exceptions import ValidationError
+from baselayer.app.access import permissions, auth_or_token
+
+from ..base import BaseHandler
+from ...models import GWDetector
+
+
+class GWDetectorHandler(BaseHandler):
+    @permissions(['Manage allocations'])
```

Comment:
Is this the right permissions?

Reply:
At this point, allocations is kind of a catch all for instrument related stuff. What do you think?

Reply:
Not critical. Should consider if we want to add another permission to cover these things.

Reply:
My vote would be to leave this permission in but basically rename it.

Reply:
(in a future PR)